core/common/src/UtcOffset.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019-2021 JetBrains s.r.o.
+ * Use of this source code is governed by the Apache 2.0 License that can be found in the LICENSE.txt file.
+ */
+
+package kotlinx.datetime
+
+import kotlinx.datetime.serializers.UtcOffsetSerializer
+import kotlinx.serialization.Serializable
+
+@Serializable(with = UtcOffsetSerializer::class)
+public expect class UtcOffset {
+    public val totalSeconds: Int
+
+    public companion object {
+        public val ZERO: UtcOffset
+        public fun parse(offsetString: String): UtcOffset
+    }
+}
+public expect fun UtcOffset(hours: Int? = null, minutes: Int? = null, seconds: Int? = null): UtcOffset
```

Comment:
This needs to be documented. The other new functions as well, but the rest of them are much more self-explanatory.